vud.theme.inc

```diff
@@ -144,71 +167,71 @@ function vud_widget_proxy($variables) {
     'link_class_up' => 'vud-link-up',
     'link_class_down' => 'vud-link-down',
     'link_class_reset' => 'vud-link-reset',
-
+    
     'link_up' => '',
     'link_down' => '',
-
+    
     'class_up' => 'up-active',
     'class_down' => 'down-active',
     'class_reset' => 'reset-inactive',
-
+    
     'show_links' => FALSE,
     'show_reset' => FALSE,
     'show_up_as_link' => FALSE,
     'show_down_as_link' => FALSE,
-
+    
     'reset_long_text' => t('Reset your vote'),
     'reset_short_text' => t('(reset)'),
   );
-
+  
   $type = $variables['type'];
   $entity_id = $variables['entity_id'];
   $tag = $variables['tag'];
   $widget_theme = $variables['widget_theme'];
   $readonly = $variables['readonly'];
-
+  
   drupal_add_library('system', 'drupal.ajax');
   ctools_add_js('ajax-responder');
   ctools_include('ajax');
   // TODO: Move this to vud-comment.
   if ($type == 'comment') {
     drupal_add_css(drupal_get_path('module', 'vud_comment') . '/vud_comment.css');
   }
-
+  
   // Search and add media files.
   vud_add_files('css', $plugin);
   vud_add_files('js', $plugin);
-
+  
   $user_vote =  0;
-
+  
   // Define access per operation, for example when the vote up operation is
   // active this means the down operation should be allowed, as the up vote was
   // already.
   $up_access = $down_access = $reset_access = FALSE;
   if (user_access('use vote up/down')) {
     $criteria = array(
-      'entity_type' => $type,
-      'entity_id' => $entity_id,
-      'tag' => $variables['tag'],
-    ) + votingapi_current_user_identifier();
+        'entity_type' => $type,
+        'entity_id' => $entity_id,
+        'tag' => $variables['tag'],
+      ) + votingapi_current_user_identifier();
     $previous_vote = votingapi_select_single_vote_value($criteria);
-
+    
```

Comment:
Spaces without a real line are against coding standards, please remove this unneeded and wrong change.